crates/miden-lib/asm/miden/active_note.masm

```diff
@@ -116,12 +116,12 @@ export.get_inputs
     # => [offset, is_active_note = 1, pad(14), dest_ptr]
 
     syscall.exec_kernel_proc
-    # => [NOTE_INPUTS_COMMITMENT, num_inputs, pad(11), dest_ptr]
+    # => [NOTE_STORAGE_COMMITMENT, num_inputs, pad(11), dest_ptr]
 
     # clean the stack
     swapdw dropw dropw
     movup.5 drop movup.5 drop movup.5 drop
-    # => [NOTE_INPUTS_COMMITMENT, num_inputs, dest_ptr]
+    # => [NOTE_STORAGE_COMMITMENT, num_inputs, dest_ptr]
```

Comment:
Below this line we should also rename inputs to storage, e.g. the comments and `exec.write_storage_to_memory`.